corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -159,18 +159,22 @@ define([
         };
 
         self.newCaseProperty = function () {
-            var prop = propertyListItem(self.newPropertyName(), false, '', self.activeCaseType());
-            prop.dataType.subscribe(changeSaveButton);
-            prop.description.subscribe(changeSaveButton);
-            prop.deprecated.subscribe(changeSaveButton);
-            self.newPropertyName('');
-            self.casePropertyList.push(prop);
+            if (_.isString(self.newPropertyName())) {
```

Comment:
It looks like newPropertyName gets set to '' rather than undefined or null. Is that case caught somewhere else?

Reply:
thanks: 741d641

Reply:
Ha, I'm still concerned about that solution since I think an input field defaults to `""` as well.

Reply:
the input field defaults to `undefined`. I did test this locally.